client/my-sites/plugins-wpcom/plugin-panel.jsx

```diff
@@ -63,10 +80,11 @@ export const PluginPanel = ( {
 
 const mapStateToProps = state => {
 	const selectedSite = getSelectedSite( state );
+	const selectedSiteId = getSelectedSiteId( state );
```

Comment:
Can we just get the id from `selectedSite` or is it possible that we will have the `id` before the `site` is available?